chanzuckerberg/idseq-web
# Fix error display on forgot password page

# Description

![image](https://user-images.githubusercontent.com/28797/56060109-b349c580-5d1a-11e9-8505-97149d9df33e.png)

In debugging the missing emails, we found that this form did not display errors correctly. **NOTE** standard security practice is to **NOT** show "not found" errors on forgot password to avoid leaking membership info, so we may want to do something else. 

In the short term, it may help distinguish between missing emails and users that failed to enter their registered email correctly. 

# Test
1. go to forgot password page in incognito
2. submit "asdf@asdf.com" 
3. see error message
4. submit "gdingle@chanzuckerberg.com"
5. see no error
